components/arm/trossen/trossen.go

```diff
@@ -416,6 +487,10 @@ func (a *Arm) SleepPosition(ctx context.Context) error {
 	a.JointTo("Wrist", 2509, sleepWait)
 	a.JointTo("Forearm_rot", 2048, sleepWait)
 	a.JointTo("Elbow", 3090, sleepWait)
+	if err := a.OpenGripper(ctx); err != nil {
+		a.moveLock.Unlock()
+		return err
+	}
 	a.moveLock.Unlock()
```

Comment:
Should/could this just be deferred right after the Lock()?

Reply:
yes, it could, will change

Reply:
oh no, wait, can't do this because waitForMovement would be blocked on the same lock if there isn't an error